src/components/data/lang/en.json

```diff
@@ -27,38 +27,38 @@
+    "wikibaseQuickstatementsPort": "Which port to use for Quickstatements?",
 
     "quickStatementsConfiguration": "Configure Quickstatements",
-    "quickStatementsConfigurationNamespaceProperty": "Which namespace you want to use for properties in Wikibase?",
-    "quickStatementsConfigurationNamespaceItem": "Which namespace you want to use for items in Wikibase?",
-    "quickStatementsConfigurationPrefixProperty": "Which prefix you want to use for properties in Wikibase?",
-    "quickStatementsConfigurationPrefixItem": "Which prefix you want to use for items in Wikibase?",
+    "quickStatementsConfigurationNamespaceProperty": "Which namespace do you want to use for properties in Wikibase?",
+    "quickStatementsConfigurationNamespaceItem": "Which namespace do you want to use for items in Wikibase?",
+    "quickStatementsConfigurationPrefixProperty": "Which prefix do you want to use for properties in Wikibase?",
+    "quickStatementsConfigurationPrefixItem": "Which prefix do you want to use for items in Wikibase?",
     "quickStatementsConfigurationNamespacePropertyHelp": "",
     "quickStatementsConfigurationPrefixPropertyHelp":"",
     "quickStatementsConfigurationNamespaceItemHelp":"",
     "quickStatementsConfigurationPrefixItemHelp":"",
 
     "wikibaseConfiguration": "Configure Wikibase",
     "mediawikiConfiguration": "MediaWiki",
-    "mediawikiSiteName": "What would you like your site to be called?",
-    "mediawikiAdminName": "What would you like the username of your admin account to be?",
-    "mediawikiAdminPassword": "What would you like the password of your admin account to be?",
+    "mediawikiSiteName": "How would you like to name your MediaWiki?",
+    "mediawikiAdminName": "How would you like to name your MediaWiki admin account?",
+    "mediawikiAdminPassword": "Provide your admin account with a password.",
 
-    "databaseConfiguration": "MariaDB (MySQL)",
-    "databasePort": "Which port should the database use?",
-    "databaseName": "What should your database be called?",
-    "databaseUser": "What should the name of your database user be?",
-    "databaseUserPassword": "What should the database user password be?",
+    "databaseConfiguration": "Database (MySQL)",
+    "databasePort": "Which port to use for the database?",
+    "databaseName": "How would you like to name the database?",
+    "databaseUser": "What is the name of your database user?",
+    "databaseUserPassword": "Provide your database user with a password.",
 
-    "finish": "Download docker-compose.yml file",
-    "finish_description": "<p>You can download your docker-compose.yml file using the button to the right.</p><p>You can now run your Wikibase docker-compose environment.</p><p>To learn how to do this, refer to the <a target=\"_blank\" href=\"https://github.com/wmde/wikibase-docker/blob/master/README-compose.md\">manual</a></p>"
+    "finish": "Download Docker Compose file",
```

Comment:
See, here you say docker compose file, not wikibase-docker yml :/

Reply:
Sure. So you want to skip the description?